src/aro/Parser.zig

```diff
@@ -8448,7 +8448,7 @@ fn fixedSizeInt(p: *Parser, base: u8, buf: []const u8, suffix: NumberSuffix, tok
         const max_int = try Value.maxInt(res.ty, p.comp);
         if (interned_val.compare(.lte, max_int, p.comp)) break;
     } else {
-        res.ty = .{ .specifier = .ulong_long };
+        res.ty = .{ .specifier = if (p.comp.langopts.emulate == .gcc) .int128 else .ulong_long };
```

Comment:
This needs to check that the target supports `__int128` with `target_utils.hasFloat128`.

Reply:
hasFloat128??

Reply:
Sorry, `target_utils.hasInt128`. It looks like there is a helper for `hasFloat128` in `Compilation`, you could add the same for `hasInt128`.

Reply:
Done